### DISCUSSION TITLE
Ethiopian channels

### DISCUSSION THREAD

**Discussion by Eyob111** [iptv-org/community#39] [Q&A] [answered]
hey guys, can you do Ethiopian channels? they are open source also you can find them on the streaming websites but I couldn't make them stable.

**Comment by BellezaEmporium** [comment, marked as answer]
Basically those who are discussed in https://github.com/iptv-org/iptv/issues/3900 ? I *could* revive them back, but I shall see with @dct-infra if he's alright with hosting the code that made it possible. He's the one that hosts my version of query-streamlink right now.

**Reply by Eyob111** [reply]
cool I will wait for you.
